docs/known_issues.rst

```diff
@@ -2,5 +2,19 @@ Known issues
 ############
 
 
- **AttributeError: 'MultiDataLoader' object has no attribute 'sampler'** : This bug has been fixed in pytorch-lightning only recently and appears only in the multi-gpu mode. Quickest fix (for now) is to use a single GPU or revert to version 0.7.5 with ``pip install pytorch-lightning==0.7.5``.
+AttributeError: 'MultiDataLoader' object has no attribute 'sampler':
+--------------------------------------------------------------------
 
+This bug has been fixed in pytorch-lightning only recently and appears only in the multi-gpu mode. Quickest fix (for now) is to use a single GPU or revert to version 0.7.5 with ``pip install pytorch-lightning==0.7.5``.
+
+
+TypeError: 'NoneType' object is not iterable
```

Comment:
Actually "TypeError: 'NoneType' object is not iterable" is a quite generic error message. It is often raised because of a previous error that is the actual root cause. For instance, we had it in multi-gpu mode with the "no device" bug fixed with get_metrics_from_parameter_dict method.
The relevant error is the one below ("forward() takes 2 positional arguments but 3 were given").

Reply:
Still, this might be the error people look for in known issues.
2 options:
1 - I keep this as a title and prepare for multiple subsections with the "actual" errors.
2 - I change the title to the actual error and put the None type error in the comments.
From your comment I infer you'd prefer option 2. Do you confirm?

Reply:
Yes, I prefer option 2. But option 1 is fine too.

Reply:
updated with option 2.